ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogWorker.java

```diff
@@ -176,7 +177,7 @@ synchronized void updateIndex(long i) {
   private final RaftServer.Division server;
   private int flushBatchSize;
 
-  private Timestamp lastFlush;
+  private Timestamp outStreamLastFlush;
```

Comment:
State machine and out stream should either flush together or not flush at all.  We should not try to separate them.  So let's don't rename this field.

Reply:
The lastFlush Timestamp is useless now, I removed it